packages/kbn-config-schema/src/types/type.ts

```diff
@@ -98,6 +103,10 @@ export abstract class Type<V> {
     return this.internalSchema;
   }
 
+  public getSchemaStructure() {
+    return recursiveGetSchemaStructure(this.internalSchema);
+  }
```

Comment:
We need to expose the schema structure to the rendering service so it can traverse the schema config key paths.

Seems that the cleanest way to do this is to add a `getSchemaStructure` API to the root schema type like this.